recipes/dlib/win_boost.patch

```diff
@@ -0,0 +1,44 @@
+--- dlib/add_python_module
++++ dlib/add_python_module
+@@ -34,37 +34,16 @@
+ get_filename_component(PYTHON_PATH ${PYTHON_PATH} PATH)
+ set(CMAKE_PREFIX_PATH ${PYTHON_PATH})
+ 
+-
+-# To avoid dll hell, always link everything statically when compiling in
+-# visual studio.  This way, the resulting library won't depend on a bunch
+-# of other dll files and can be safely copied to someone elese's computer
+-# and expected to run.
+-if (MSVC)
```

Comment:
I get unending amounts of email from users who do not, in any sense,
understand the distinction between static and dynamic linking, or even what
linking is at all.  If the default in dlib's files isn't to statically link
I will get even more emails about "dlib runtime errors" that are because
people have dynamically linked to dlls that are not available when they run
python because they have misconfigured their systems, not because of any
dlib related problems.  So I'm not super excited to change this.